boards/arm/serpente/serpente_defconfig

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_SAMD21=y
+CONFIG_SOC_PART_NUMBER_SAMD21E18A=y
+CONFIG_BOARD_SERPENTE=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_SAM0=y
```

Comment:
The SAM0 drivers are now automatically enabled when the corresponding feature is set, so you can keep the `CONFIG_SERIAL=y` etc and drop the `CONFIG_UART_SAM0=y` etc.